cg/store/filters/status_order_filters.py

```diff
@@ -9,12 +9,6 @@
 from cg.store.models import Customer, Order
 
 
-def filter_orders_by_workflow(orders: Query, workflow: str | None, **kwargs) -> Query:
-    if workflow == Workflow.BALSAMIC:
-        return orders.filter(Order.workflow.startswith(workflow))
```

Comment:
We should test how this functionality works with this change. This ensured that balsamic-umi and balsamic-qc orders shows up under the balsamic tab in cigrid. We might need to add something similar for the get_orders endpoint

Reply:
I moved this logic part to the order service.